src/seedu/addressbook/ui/TextUi.java

```diff
@@ -165,5 +166,16 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
-
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   String getPrintableString(Printable... printables){
+	   StringBuilder printableResult = new StringBuilder();
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.